### PR TITLE
cc110x: fix typo

### PR DESCRIPTION
**Pull request by OlegHahm**
Got apparently never built.

### CODE REVIEW

**Comment by cgundogan** [comment]
ACK please squash

File: drivers/cc110x/cc110x.c [hunk 1]
```diff
@@ -87,14 +87,11 @@ int cc110x_setup(cc110x_t *dev, const cc110x_params_t *params)
...
 
     /* set default node id */
-#ifdef CPUID_LEN
-    if (CPUID_LEN > 0) {
-        char cpuid[CPUID_LEN];
-        cpuid_get(cpuid);
-        for (int i = 1; i < CPUID_LEN; i++) {
-            cpuid[0] ^= cpuid[i]
-        }
-        cc110x_set_address(dev, (uint8_t) cpuid[0]);
```
Comment (kaspar030) [inline drivers/cc110x/cc110x.c]: Not setting the address anymore?

Comment (OlegHahm) [inline drivers/cc110x/cc110x.c]: fixed

**Comment by OlegHahm** [comment]
squashed

**Comment by cgundogan** [comment]
How did you pay for that.. (:

**Comment by OlegHahm** [comment]
You don't wanna know.

**Pull request merged**